test/unit/src/lib/utils/firebase-loader.js

```diff
@@ -0,0 +1,29 @@
+const {assert} = require('../assert');
+const loader = require('../../../../../src/lib/utils/firebase-loader');
+
+describe('firebase-loader', function() {
```

Comment:
is it worth exporting `internalPreload` so we can test that it does the things we expect it to do?

Reply:
I could be convinced, but I don't think so.